recipes/chxtools/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "chxtools" %}
+{% set version = "3.0.2" %}
+{% set sha256 = "1d6bb5516c413f85f6502850b4e5563dc899d682adeb4aa242e1525ba59efce1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/NSLS-II-CHX/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.6
+    - databroker
+    - eiger-io
+    - matplotlib
+    - numpy
+    - pyepics
+    - pymongo
+    - scikit-beam
+    - scikit-image
+    - scipy
+
+test:
+  imports:
+    - chxtools
```

Comment:
Are there better import tests to add to actually test the package is good?

Reply:
I added more imports in https://github.com/nsls-ii-forge/chxtools-feedstock/pull/3. @kbeeperez, please use the imports from my PR for `chxtools` here.